src/kedro_vizro_labeling/dashboards/components.py

```diff
@@ -212,6 +217,8 @@ def _make_page_layout(self, *args, **kwargs):
         if self.missing_permission_modal is not None:
             additional_build_obj.append(self.missing_permission_modal.build())
 
+        # AM comment: super_build_obj["settings"].children.extend(additional_build_obj) should do it without needing
+        # theme_switch anywhere.
```

Comment:
Indeed, thanks a lot!

Reply:
Actually, just noticed `super_build_obj["settings"].children` is not a list, so I think I have to do it that way.